src/app/ApplicationTemplate.Shared.Views/Configuration/ApiConfiguration.cs

```diff
@@ -42,14 +39,6 @@ private static IServiceCollection AddMainHandler(this IServiceCollection service
 
 	private static Task<bool> GetIsNetworkAvailable(CancellationToken ct)
 	{
-//-:cnd:noEmit
-#if WINDOWS_UWP || __ANDROID__ || __IOS__
-		// TODO #172362: Not implemented in Uno.
-		// return NetworkInformation.GetInternetConnectionProfile()?.GetNetworkConnectivityLevel() == NetworkConnectivityLevel.InternetAccess;
-		return Task.FromResult(Xamarin.Essentials.Connectivity.NetworkAccess == Xamarin.Essentials.NetworkAccess.Internet);
-#else
-		return Task.FromResult(true);
-#endif
-//+:cnd:noEmit
+		return Task.FromResult(NetworkInformation.GetInternetConnectionProfile()?.GetNetworkConnectivityLevel() == NetworkConnectivityLevel.InternetAccess);
```

Comment:
shouldn't this use your newly added `IConnectivity`? That way it could be mocked.